config/initializers/rollbar.rb

```diff
@@ -0,0 +1,52 @@
+require 'rollbar/rails'
+Rollbar.configure do |config|
+  # Without configuration, Rollbar is enabled in all environments.
+  # To disable in specific environments, set config.enabled=false.
+  
+  config.access_token = ENV['ROLLBAR_ACCESS_TOKEN']
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.